ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/activity-picklist-remote-auto-complete-options.component.ts

```diff
@@ -125,7 +125,7 @@ export class ActivityPicklistRemoteAutoCompleteOptionsComponent
     ): ActivityPicklistOption {
         return {
             ...otherOption,
-            optionLabel: this.searchValue$.value.toLocaleUpperCase(),
+            optionLabel: this.searchValue$.value,
```

Comment:
Please exclude leading and trailing whitespace.

Reply:
Sure